src/docs/AppCanvas.vue

```diff
@@ -19,6 +19,57 @@ const fontFamilies = [
   'Verdana',
 ];
 
+const fontSpecificOffsets = {
+  'Comic Sans MS': {
+    underline: -2,
+    strikethrough: -4,
+  },
+  'Times New Roman': {
+    underline: 0,
+    strikethrough: 0,
+  },
+  'Courier New': {
+    underline: 1,
+    strikethrough: -1,
+  },
+  Georgia: {
+    underline: -1,
+    strikethrough: 0,
+  },
+  Impact: {
+    underline: 0,
+    strikethrough: 0,
+  },
+  Inter: {
+    underline: 0,
+    strikethrough: -2,
+  },
+  Montserrat: {
+    underline: 0,
+    strikethrough: 0,
+  },
+  Roboto: {
+    underline: 0,
+    strikethrough: 0,
+  },
+  Verdana: {
+    underline: -1,
+    strikethrough: 5,
+  },
+};
+
+function getSmartOffset(fontFamily, type) {
+  const offsets = fontSpecificOffsets[fontFamily];
+  if (!offsets) return 0;
+
+  if (type === 'underline') {
+    return offsets.underline !== undefined ? offsets.underline : 0;
+  } else if (type === 'strikethrough') {
+    return offsets.strikethrough !== undefined ? offsets.strikethrough : 0;
+  }
+  return 0;
+}
```

Comment:
This is not how I would like to do the demo. By doing this, you are duplicating logic that is already in the library, and so making it look like it's demo that knows about the various font-specific offsets. This won't scale well either when we add support for more fonts over time. We'll trying to test the offsets and wondering why the offsets are never being applied -- because the demo is overriding them for fonts it doesn't know about.

That's why the number inputs need to support a "cleared" state where the offsets are not specified at all in the config, so that the library can take care of applying the offsets.